.github/workflows/ci.yml

```diff
@@ -8,15 +8,15 @@ on:
 jobs:
   build:
     name: Build
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
you can use `ubuntu-latest` to avoid this issue in the future

Reply:
nice, thanks :)